source/includes/_plugin_integrations_configuration_form_notes.md

```diff
@@ -0,0 +1,81 @@
+### Integration Configuration Form Notes
+
+The integration framework lets developer define their custom messages for the plugin's configuration form. 
+
+The `ConfigSupport` class should implement the `\Mautic\IntegrationsBundle\Integration\Interfaces\ConfigFormNotesInterface`.
+
+```php
+<?php
+
+declare(strict_types=1);
+
+namespace MauticPlugin\HelloWorldBundle\Integration\Support;
+
+use Mautic\IntegrationsBundle\DTO\Note;
+use Mautic\IntegrationsBundle\Integration\ConfigFormNotesTrait;
+use Mautic\IntegrationsBundle\Integration\DefaultConfigFormTrait;
+use Mautic\IntegrationsBundle\Integration\Interfaces\ConfigFormAuthInterface;
+use Mautic\IntegrationsBundle\Integration\Interfaces\ConfigFormFeatureSettingsInterface;
+use Mautic\IntegrationsBundle\Integration\Interfaces\ConfigFormFeaturesInterface;
+use Mautic\IntegrationsBundle\Integration\Interfaces\ConfigFormInterface;
+use Mautic\IntegrationsBundle\Integration\Interfaces\ConfigFormNotesInterface;
+use Mautic\IntegrationsBundle\Integration\Interfaces\ConfigFormSyncInterface;
+use Mautic\IntegrationsBundle\Mapping\MappedFieldInfoInterface;
+use Mautic\IntegrationsBundle\Sync\SyncDataExchange\Internal\Object\Company;
+use Mautic\IntegrationsBundle\Sync\SyncDataExchange\Internal\Object\Contact;
```

Comment:
Although the use statements are quite long and for example this one is not used. Can you remove the unused ones?